src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,141 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import { buildHybridComponent } from '../../util/state-management';
```

Comment:
Let's adjust this to use `buildModernHybridComponent` instead. See ToolTip for an example of how to use it.

Reply:
I think this may have been reviewed before I updated it to use `buildModernHybridComponent`, but it does use it now